src/org/parosproxy/paros/extension/history/ExtensionHistory.java

```diff
@@ -891,6 +892,14 @@ public boolean supportsDb(String type) {
     	return true;
     }
 
+	/**
+	 * @since TODO
+	 */
+	//ZAP: add getter to expose UI component of History tab

```

Comment:
These should be added after the license header (we no longer add the ZAP comments in the middle of the code).